scripts/dev/configure-debian

```diff
@@ -8,10 +8,10 @@
 SCRIPTS_DIR=$(dirname "$0")
 . "${SCRIPTS_DIR}/configure-common"
 
-cd $BUILD
+cd "$BUILD" || exit
 cmake \
 	-DPLUGINS="ALL" \
 	-DTOOLS="ALL" \
 	-DENABLE_DEBUG="OFF" \
 	-DENABLE_LOGGER="OFF" \
-	$*
+	"$*"
```

Comment:
This cannot work, e.g. if ~e is the path to Elektra following does not work anymore: `~e/scripts/dev/configure-debian -DCMAKE_INSTALL_PREFIX="/home/installation" -DTOOLS="ALL" -DBINDINGS="ALL" ~e`

Reply:
I found the following occurrences of this problem:

https://github.com/ElektraInitiative/libelektra/pull/4601/commits/565379e36fe6bfbe818f8004b33d80ea27c583a8

(Ironically this includes the example @kodebach and I discussed earlier in this PR...)

One thing I am not sure about and don't know how to test are the scripts `scripts/docker/jenkinsnode/*`, so please have an extra careful look at those changes.

Reply:
> One thing I am not sure about and don't know how to test are the scripts `scripts/docker/jenkinsnode/*`, so please have an extra careful look at those changes.

I think you should be able to run those in your build directory. AFAICT they are used by the Jenkins CI.

Reply:
I managed to build Elektra in the Jenkis node container with the modified scripts. So I think the changes there are fine.